src/components/storefront/ProductDetails/ProductDetails.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Heading, Flex, Image, Text, Button } from '@chakra-ui/react';
+import AdditionalContent from './AdditionalContent';
+
+const ProductDetails = ({ product }) => {
+  const { images, title, description, variants } = product;
+  const productSku = variants && variants[0].sku;
+  const productImage = images && images[0].src;
+  const productPrice = variants && variants[0].price;
```

Comment:
in this particular component, `variants` and `images` should always be defined right? so no need for the `&&`

Reply:
I think the answer is yes (for most cases). Product is passed as a prop and I have yet to see a product that has `product.variants.length < 1` - in the event that there is for some reason, this would break our app. So it mostly depends on our Shopify store, as long as it's being used correctly the && isn't necessary. What are your thoughts?

```
const ProductDetails = ({ product }) => {
  const { images, title, description, variants } = product;

Reply:
yep, let's remove the `&&`